requirements/base.txt

```diff
@@ -6,3 +6,4 @@ requests>=1.0.0
 tornado>=4.2.1
 # Required by Tornado to handle threads stuff.
 futures>=2.0
+pycurl>=7.19.3
```

Comment:
We can certainly find a comprimise here. I recommend that we keep these configuration options while issuing a warning to the user on start that the necessary libraries be installed in order to facilitate this functionality. Salt's http utility library should be made to work in either case, preferring the pycurl implementation if it is installed on the system.